onnxruntime/contrib_ops/rocm/bert/skip_layer_norm_tunable_op.h

```diff
@@ -54,30 +56,50 @@ Status SkipLayerNormSmallOp(const SkipLayerNormParams<T>* params) {
   return HIP_CALL(hipGetLastError());
 }
 
-#define ADD_OP(threads_per_block)                                         \
-  this->ops_.emplace_back(SkipLayerNormSmallOp<T, threads_per_block, 1>); \
-  this->ops_.emplace_back(SkipLayerNormSmallOp<T, threads_per_block, 2>); \
-  this->ops_.emplace_back(SkipLayerNormSmallOp<T, threads_per_block, 4>); \
-  this->ops_.emplace_back(SkipLayerNormSmallOp<T, threads_per_block, 8>); \
-  this->ops_.emplace_back(SkipLayerNormSmallOp<T, threads_per_block, 16>);
+template <typename T, int ThreadsPerBlock, int VecSize>
+Status SkipLayerNormRegularOp(const SkipLayerNormParams<T>* params) {
```

Comment:
please be careful about rocm EP, our pipeline is not fixed for rocm EP yet, so you can't verify the build in pipeline. 
please make sure those files are identity to main branch as we haven't work on AMD gpu yet.

Reply:
@Peixuanzuo and I are actively working on it. The CI will soon be enabled for ROCm. Drop us a msg if you have any question.